zuluCrypt-cli/bin/open_volume.c

```diff
@@ -362,6 +362,27 @@ int zuluCryptEXEOpenVolume( const struct_opts * opts,const char * mapping_name,u
 		/*
 		* zuluCryptCreateMountPoint() is defined in create_mount_point.c
 		*/
+
+        // get mountpoint for user, not for root
+        int found = 0;
+        int found_at;
+        int ii = 0;
+        while (!found) {
```

Comment:
This still needs a way to terminate the while loop if we make it to the end of the the opts->env array.